esyi/FetchStorage.ml

```diff
@@ -50,17 +50,8 @@ let install ~cfg ~path dist =
   let finishInstall path =
 
     let%bind () =
-      let f {Package.File. name; content; perm} =
-        let name = Path.append path name in
-        let dirname = Path.parent name in
-        let%bind () = Fs.createDir dirname in
-        (* TODO: move this to the place we read data from *)
-        let contents =
-          if String.get content (String.length content - 1) == '\n'
-          then content
-          else content ^ "\n"
-        in
-        let%bind () = Fs.writeFile ~perm ~data:contents name in
+      let f file =
+        let%bind _ = Package.File.writeToDir ~destinationDir:path file in
         return()
```

Comment:
nit: this two lines above could be just
```
Package.File.writeToDir ~destinationDir:path file
```
In general I try to avoid `let _ =` (or `let%bind _ =`) as in case we change `writeToDir` to return something meaningful rather than `unit` — we'd want to update all callsites and with `_` compiler won't tell us to update as `_` will match anything.